src/Elliptic/DiscontinuousGalerkin/CMakeLists.txt

```diff
@@ -5,6 +5,12 @@ set(LIBRARY EllipticDg)
 
```

Comment:
Perhaps considering changing the commit subject to use the word 'Move' rather than 'Add', as it currently makes it seem like this commit introduces the function, though the second line clarifies.